common/config/dpaths.ts

```diff
@@ -48,5 +48,19 @@ export const ETH_SINGULAR: DPath = {
   value: "m/0'/0'/0'"
 };
 
+// Full length deterministic wallet paths from BIP44
+// https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki
+// normal path length is 4, ledger is the exception at 3
+
+// m / purpose' / coin_type' / account' / change / address_index
+//   |          |            |          |        |
+//   | constant |   index    |  index   | 0 or 1 |
+//   |__________|____________|__________|________|
+
+// whitespace strings are evaluated the same way as nospace strings, except they allow optional spaces between each portion of the string
+// ie. "m / 44' / 0' / 0'" is valid, "m / 4 4' / 0' / 0'" is invalid
+export const dPathRegex = /m\/44'\/[0-9]+\'\/[0-9]+(\'+$|\'+(\/[0-1]+$))/;
+export const whitespaceDPathRegex = /m\s*\/\s*44'\s*\/\s*[0-9]+\'\s*\/\s*[0-9]+(\'+$|\'+\s*(\/\s*[0-1]+$))/;
```

Comment:
Does this get used in code anywhere, other than the unit tests? No use in testing code that doesn't actually run in the product.

Reply:
No, it's not actually used anywhere right now. It doesn't need to be tested, since currently it isn't used. It'd probably be better just to comment it out, if we ever plan to take user input and want to verify BIP44 strings, it'd be good to accept strings with whitespace.

Reply:
Ah, looks like we disabled custom paths in DeterministicWalletsModal.tsx. That's where we were allowing user input for these. But the code is still all there for it.